scripts/microbe_stage/setup.as

```diff
@@ -363,6 +363,20 @@ void cellHitAgent(GameWorld@ world, ObjectID firstEntity, ObjectID secondEntity)
 
 }
 
+uint getHexCountOrganelle(PlacedOrganelle@ organelle)
+{
+    return organelle.organelle.getHexes().length();
+}
+
+uint getTotalHexes(array<PlacedOrganelle@> organelles)
```

Comment:
This parameter should be taken in as a reference.

Reply:
And this function should be in a more sensible file. Like organelle.as or organelle_table.as

Reply:
I would suggest caching the hex number somewhere instead of calling the function whenever its needed.

Reply:
Okay, I'll try to dive deeper in the angelscripts to find where and how I cache it.

The biggest reason I kept the functions so local was because it created the least opportunity for error, good to hear the logic parts I threw together seem to be okay though.